frontend/database/00200_create_runs_groups.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `Runs_Groups` (
+  case_run_id int NOT NULL AUTO_INCREMENT,
```

Comment:
```suggestion
  run_group_id int NOT NULL AUTO_INCREMENT,
```

todos los IDs corresponden con el nombre de la tabla.